dcpy/utils/versions.py

```diff
@@ -67,10 +67,11 @@ def label(self) -> str:
     def __lt__(self, other) -> bool:
         match other:
             case MajorMinor():
-                return (self.year, self.major, self.minor) < (
+                return (self.year, self.major, self.minor, self.patch) < (
```

Comment:
nit: if we just add @dataclass(order=True), we can turn this into `return self < other`. I think.

Reply:
Seems a little odd to use the function that's being overriden within here? Doesn't < make a call to __lt__ for these classes? Just seemed cleaner to not use `self <` in `self.__lt__`. though to be honest not entirely sure the way these things are handled under the hood